components/application-broker/internal/broker/testing/api_object.go

```diff
@@ -97,6 +107,34 @@ func NewDefaultBroker(ns string) *eventingv1alpha1.Broker {
 	}
 }
 
+func NewIstioPolicy(ns, policyName string) *istiov1alpha1.Policy {
+	brokerTargetSelector := &istioauthenticationalpha1.TargetSelector{
+		Name: brokerTargetSelectorName,
+	}
+	filterTargetSelector := &istioauthenticationalpha1.TargetSelector{
+		Name: filterTargetSelectorName,
+	}
+	mtls := &istioauthenticationalpha1.MutualTls{
+		Mode: istioMtlsPermissiveMode,
+	}
+	peerAuthenticationMethod := &istioauthenticationalpha1.PeerAuthenticationMethod{
+		Params: &istioauthenticationalpha1.PeerAuthenticationMethod_Mtls{
+			Mtls: mtls,
+		},
+	}
+
+	return &istiov1alpha1.Policy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      policyName,
+			Namespace: ns,
```

Comment:
Let's assign some coherent labels.

Reply:
@sayanh Would you please suggest examples?

Reply:
How about `eventing.knative.dev/broker: default`? Rationale is the policy is for the `default` broker